app/assets/stylesheets/chat.scss

```diff
@@ -106,10 +106,12 @@
 
 .chat__channels input {
   width: 100%;
+  outline: none;
 }
```

Comment:
If the outline is removed, there needs to be some other visual indicator for keyboard users/accessibility.